pkg/util/command_util.go

```diff
@@ -321,7 +322,44 @@ Loop:
 	return nil
 }
 
-func GetUserFromUsername(userStr string, groupStr string) (string, string, error) {
+// Extract user and group id from a string formatted 'user:group'.
+// If gidEqualToUIDIfNotGiven is set, the gid is equal to uid if the group is not specified
+// otherwise gid is set to zero.
+func GetUIDAndGIDFromString(userGroupString string, gidEqualToUIDIfNotGiven bool) (uint32, uint32, error) {
+	userAndGroup := strings.Split(userGroupString, ":")
+	userStr := userAndGroup[0]
+	var groupStr string
+	if len(userAndGroup) > 1 {
+		groupStr = userAndGroup[1]
+	}
+
+	uidStr, gidStr, err := GetUserFromUsername(userStr, groupStr, !gidEqualToUIDIfNotGiven)
+	if err != nil {
+		return 0, 0, err
```

Comment:
Making these constants would also be great

Reply:
These both zeros are only dummy values, as somewhat needs to be set to return in the err case. They are not used (and should not) if err is set.
What would be best practise here?